### DISCUSSION TITLE
How do you add the icon?

### DISCUSSION THREAD

**Discussion by cvitacek** [drivardxrm/FluentUI.Slider.PCF#3] [General]
Nice control, useful in lots of ways, however I can't seem to figure out how to add the "star" icon shown in all your examples.

**Comment by drivardxrm** [comment]
Hi @cvitacek ,

The star that I show in my example is an emoji like this one : ⭐
You need to add the star in the '**suffix**' parameter when you configure the control


![image](https://github.com/drivardxrm/FluentUI.Slider.PCF/assets/38399134/103f3dbc-0090-49d6-b3f7-3d4791768d98)

![image](https://github.com/drivardxrm/FluentUI.Slider.PCF/assets/38399134/8336bb49-9ba1-4ae0-9781-b9c7807f5097)
